applications/sandbox/systems/testsystem.hpp

```diff
@@ -111,6 +112,9 @@ class TestSystem final : public System<TestSystem>
             delete[] buf;
         }
 
+        std::cout << "Yay";
+        std::cout << "please run";
```

Comment:
we have a logging system

Reply:
Just added it so I could attempt the analyze/build actions to even notice my changes. but thats fixed now